cmd/diff-main.go

```diff
@@ -34,24 +34,19 @@ var (
 
 // Compute differences between two files or folders.
 var diffCmd = cli.Command{
-	Name:        "diff",
-	Usage:       "Show differences between two folders or buckets.",
-	Description: "Diff only lists missing objects or objects with size differences. It *DOES NOT* compare contents. i.e. Objects of same name and size, but differ in contents are not noticed.",
-	Action:      mainDiff,
-	Before:      setGlobalsFromContext,
-	Flags:       append(diffFlags, globalFlags...),
+	Name:   "diff",
+	Usage:  "Show differences between two folders or buckets. Diff only lists missing objects or objects with size differences. It *DOES NOT* compare contents. i.e. Objects of same name and size, but differ in contents are not noticed.",
```

Comment:
@balamurugana, Description section is being removed here to make it consistent with other mc commands.